biogeochem/EDCohortDynamicsMod.F90

```diff
@@ -381,11 +392,25 @@ subroutine InitPRTBoundaryConditions(new_cohort)
 
     case (prt_cnp_flex_allom_hyp)
 
-       write(fates_log(),*) 'You have not specified the boundary conditions for the'
-       write(fates_log(),*) 'CNP with flexible stoichiometries hypothesis. Please do so. Dude.'
-       call endrun(msg=errMsg(sourcefile, __LINE__))
+       call new_cohort%prt%RegisterBCIn(acnp_bc_in_id_pft,bc_ival = new_cohort%pft)
```

Comment:
These function calls have evolved to become quite complex. Can you add some sort of guide to what the function is doing so help the uninitiated navigate the code?

Reply:
By which, I mean a couple of sentences of comment here, rather than a huge guidebook ;)

Reply:
I updated the code  a little bit. At the head of this routine, there should be some concise descriptive text about what is going on here.